lib/syskit/roby_app/remote_processes/protocol.rb

```diff
@@ -12,13 +12,15 @@ module RemoteProcesses
             COMMAND_END        = "E"
             COMMAND_QUIT       = "Q"
             COMMAND_KILL_ALL   = "K"
+            COMMAND_WAIT_RUNNING = "W"
             COMMAND_LOG_UPLOAD_FILE  = "U"
             COMMAND_LOG_UPLOAD_STATE = "X"
 
             EVENT_DEAD_PROCESS = "D"
             RET_STARTED_PROCESS = "P"
             RET_YES = "Y"
             RET_NO  = "N"
+            RET_RUNNING = "R"
```

Comment:
how do these commands work?

Reply:
But you reminded me to remove the RET_RUNNING one, as it is not necessary anymore

Reply:
> It's just a char that identifies the command the client is requesting the server to perform.

Does it send a char via udp?

Reply:
TCP